test/integ/quote-classic.test.ts

```diff
@@ -2239,6 +2373,7 @@ describe('quote', function () {
   };
 
   // TODO: Find valid pools/tokens on optimistic kovan and polygon mumbai. We skip those tests for now.
+  // TODO: Would love to test portion on other supported production chains, but don't know how much effort to make that happen
```

Comment:
its fine to leave this out for now, can fast follow if we have portion configured on the other chains

Reply:
Yea historically we haven't tested executeSwap on non mainnet chains, and I don't see a need to do that for portion specific tests, but we can at least test the quote response object here.